test/e2e/base_user_test.go

```diff
@@ -91,3 +96,57 @@ func newBannedUser(host *wait.HostAwaitility, email string) *v1alpha1.BannedUser
 		},
 	}
 }
+
+func (s *baseUserIntegrationTest) deactivateAndCheckUser(userSignup *v1alpha1.UserSignup, mur *v1alpha1.MasterUserRecord) {
+	userSignup, err := s.hostAwait.UpdateUserSignupSpec(userSignup.Name, func(us *v1alpha1.UserSignup) {
+		us.Spec.Deactivated = true
+	})
+	require.NoError(s.T(), err)
+	s.T().Logf("user signup '%s' set to deactivated", userSignup.Name)
+
+	err = s.hostAwait.WaitUntilMasterUserRecordDeleted(mur.Name)
+	require.NoError(s.T(), err)
+
+	// "deactivated"
+	notifications, err := s.hostAwait.WaitForNotifications(userSignup.Status.CompliantUsername, v1alpha1.NotificationTypeDeactivated, 1, wait.UntilNotificationHasConditions(Sent()))
+	require.NoError(s.T(), err)
+	require.NotEmpty(s.T(), notifications)
+	require.Equal(s.T(), 1, len(notifications))
```

Comment:
you could just use `require.Len(s.T(), notifications, 1)` here, but it's just a suggestion

Reply:
That's true, changed it. 0833462a01b43b3b02ead7cc8f8a76429dd89ab4